LoopKitUI/Views/Settings Editors/SuspendThresholdEditor.swift

```diff
@@ -159,7 +162,12 @@ public struct SuspendThresholdEditor: View {
     }
 
     private var saveButtonState: ConfigurationPageActionButtonState {
-        initialValue == nil || value != initialValue || mode == .acceptanceFlow ? .enabled : .disabled
+        let selectableValues = picker.selectableValues
+        let adjustedBounds = (selectableValues.first!)...(selectableValues.last!)
+        guard adjustedBounds.contains(value.doubleValue(for: displayGlucoseUnitObservable.displayGlucoseUnit)) else {
+            return .disabled
+        }
```

Comment:
[nit] why not use the guardrail bounds here, since they are input to the picker?

Reply:
Because the user selectable bounds actually change, based on unit conversion, to the nearest hundredth of mmol/L when the user is in mmol/L.  The user can select a value outside the mg/dL bounds, in this case, and be prevented from continuing, when they should be allowed.